test/specs/carousel.test.js

```diff
@@ -494,22 +494,24 @@ describe('<Carousel />', () => {
         expect(wrapper).toHaveState({ currentSlide: 1 });
       });
 
-      it('should not move to next slide when pressing right arrow key', () => {
-        const map = {};
-        document.addEventListener = jest.fn((event, cb) => {
-          map[event] = cb;
+      describe('when disableKeyboardControls is set to true', () => {
```

Comment:
sorry to make this confusing! we don't need another `describe` block, I just meant the assertion (`it...does some thing`) was more descriptive.

Reply:
so it would look like `it('should not move to next slide when pressing right arrow key when disableKeyboardControls is set to true', () => {`

Reply:
has this been resolved? @ElreyB